packages/core/src/client.ts

```diff
@@ -4,22 +4,30 @@ import { OrganizationController } from './controllers/organization';
 import { UserController } from './controllers/user';
 import { FileController } from './controllers/file';
 import { AppController } from './controllers/app';
-import { PaginationMetadata } from '@slashauth/types';
 
-type ResponseMeta<T> = Omit<rm.IRestResponse<T>, 'result'>;
-type ErrorMessage = any;
+type ErrorMessage = string;
```

Comment:
Is there a world where we want the error to be an object so we can encode an `errorCode` and `errorMessage`? I've seen this done before and we do it in a few places today

Reply:
is errorCode the same as the http statusCode or something else?

Reply:
StatusCode is the HTTP status code. ErrorCode is something that we can define for easier communciation with clients. Parsing error messages to determine the error is generally a bad idea so most protocols develop an error code (either a number or a string, depending on their implementation)